engine/lib/memcache.php

```diff
@@ -43,15 +24,33 @@ function is_memcache_available() {
  *
  * @return void
  * @access private
+ * @since 2.1
+ *
+ * @see login() Called in login() so that last action is updated on next request
```

Comment:
This function has been removed from login() by you... was that intended? Does this `@see` need updating?